content/guides/core-concepts/cypress-studio.md

```diff
@@ -229,12 +276,17 @@ describe('Cypress Studio Demo', () => {
 
 ##### <Icon name="graduation-cap"></Icon> Real World Example
 
-Clone the <Icon name="github"></Icon> [Real World App (RWA)](https://github.com/cypress-io/cypress-realworld-app) and refer to the [cypress/tests/demo/cypress-studio.spec.ts](https://github.com/cypress-io/cypress-realworld-app/blob/develop/cypress/tests/demo/cypress-studio.spec.ts) file.
+Clone the <Icon name="github"></Icon>
+[Real World App (RWA)](https://github.com/cypress-io/cypress-realworld-app) and
+refer to the
+[cypress/tests/demo/cypress-studio.spec.ts](https://github.com/cypress-io/cypress-realworld-app/blob/develop/cypress/tests/demo/cypress-studio.spec.ts)
+file.
 
 </Alert>
 
 ## History
 
 | Version                                     | Changes                              |
 | ------------------------------------------- | ------------------------------------ |
+| [8.1.0](/guides/references/changelog#8-1-0) | Added assertions to guide            |
```

Comment:
This is a history of the product changes for the feature, not changes to the docs themselves.

```suggestion
| [8.1.0](/guides/references/changelog#8-1-0) | Added ability to generate assertions            |
```

Reply:
Ah thanks, didn't know that